packages/ui-components/src/Address/Address.styles.tsx

```diff
@@ -2,12 +2,13 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import React from 'react';
+import styled from 'styled-components';
 
-export class AccountSummary extends React.PureComponent {
-  render () {
-    return (
-      'Account Summary'
-    );
+export const AddressContainer = styled.div`
+  &&& {
```

Comment:
I've never seen the &&& syntax, what does it mean?

Reply:
it's actually not necessary on this particular case, but when overriding styles from Semantic UI components, &&& makes sure the style component styles are as specific as possible so that it 'wins'. Functionally it's more or less like adding `!important` to every line.

Reply:
Got it, thanks!